src/host/cmdline.cpp

```diff
@@ -198,7 +194,8 @@ void CommandLine::EndAllPopups()
 {
     while (!_popups.empty())
     {
-        EndCurrentPopup();
+        _popups.front()->End();
+        _popups.pop_front();

```

Comment:
Why this change? What's wrong with `EndCurrentPopup`?

Reply:
Because EndCurrentPopup already does the checking, which is redundant. This should be inlined